src/main/java/io/supertokens/inmemorydb/Start.java

```diff
@@ -1720,31 +1740,22 @@ public void insertUsedCode(TOTPUsedCode usedCodeObj)
     }
 
     @Override
-    public TOTPUsedCode[] getNonExpiredUsedCodes(String userId)
+    public TOTPUsedCode[] getAllUsedCodes(String userId)
```

Comment:
You should also call the function in the plugin interface `getAllUsedCodesDescOrder`

Reply:
Updated.